### PR TITLE
Add symbolic name of the breakpoint to be edited, resolves #1326

### LINKED ISSUE #1326: put the symbolic name of the breakpoint you edit
The dialog shows `Edit breakpoint at XXXXXXXX` this should be `Edit breakpoint at kernel32.GetProcAddress`

### PR DESCRIPTION
**Pull request by ner0x652**
This change is [Reviewable](https://reviewable.io/reviews/x64dbg/x64dbg/1346)

### CODE REVIEW

File: src/gui/Src/Gui/EditBreakpointDialog.cpp [hunk 1]
```diff
@@ -15,7 +16,8 @@ EditBreakpointDialog::EditBreakpointDialog(QWidget* parent, const BRIDGEBP & bp)
     }
     else
     {
-        setWindowTitle(QString(tr("Edit Breakpoint %1")).arg(ToHexString(bp.addr)));
+        DisassemblerGraphView disGraphView;
```
Comment (ner0x652) [inline src/gui/Src/Gui/EditBreakpointDialog.cpp]: Ok, I've made a new pull request with `getSymbolicName` copied into `MiscUtils`.

**Comment by torusrxxx** [comment]
There should be a global routine shared by DBG and GUI and plugins to
convert an address to its symbolic name or label, in a uniform way. I think
_dbg_addrinfoget with flag_label is suitable for this role.

**Comment by ner0x652** [comment]
@torusrxxx @mrexodia I am agree with the idea of a global function instead of what is in the pull request. I'll change it.

**Comment by mrexodia** [comment]
Thanks!

**Pull request merged**